packages/synapse-react-client/src/lib/containers/FileUpload.tsx

```diff
@@ -9,7 +9,6 @@ export type FileUploadProps = {
   label?: string
   buttonProps?: ButtonProps
   uploadCallback?: (response: UploadCallbackResp) => void
-  context?: any // to distinguish which file was uploaded if we have several FileUpload components on the same page
```

Comment:
`context` prop is unnecessary. Each instance can have its own version of `uploadCallback` that contains this information.